src/java.desktop/share/classes/sun/print/RasterPrinterJob.java

```diff
@@ -2475,8 +2475,9 @@ public void cancel() {
     }
 
     /**
-     * Returns true is a print job is ongoing but will
-     * be cancelled and the next opportunity. false is
+     * Returns true if a print job is ongoing and user
+     * has cancelled the printjob and will
+     * be cancelled at the next opportunity. false is
      * returned otherwise.
```

Comment:
Does it add anything to the description of [`PrinterJob.isCancelled`](https://docs.oracle.com/en/java/javase/21/docs/api/java.desktop/java/awt/print/PrinterJob.html#isCancelled())?

<q cite="https://docs.oracle.com/en/java/javase/21/docs/api/java.desktop/java/awt/print/PrinterJob.html#isCancelled()">Returns `true` if a print job is in progress, but is going to be cancelled at the next opportunity; otherwise returns `false`.</q>

The description here repeats the same but in other words. I propose removing this javadoc comment altogether.

Reply:
I guess we can remove or we can put @inheritDoc..
@prrace Any preference from you?